Build/scripts/compose_build.ps1

```diff
@@ -0,0 +1,56 @@
+#-------------------------------------------------------------------------------------------------------
+# Copyright (C) Microsoft. All rights reserved.
+# Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+#-------------------------------------------------------------------------------------------------------
+
+# Compose Build script
+#
+# Aggregate metadata about a build and produce a file with useful information about the build
+# for tools to consume and get a quick overview of the status of a completed build.
+
+param (
+    [Parameter(Mandatory=$True)]
+    [string]$rootPath
+)
+
+#
+# Clean up the sentinel which previously marked this build as incomplete.
+#
+
+$buildIncompleteFile = Join-Path $rootPath "build.incomplete"
```

Comment:
Wasn't there a racing condition to use this file for all configuration ? I thought we were going for 1 file per configuration